### PR TITLE
successfully changed the Level-1/code.yml

### PR DESCRIPTION
**Pull request by Debshibraj123**
### Changes

Closes:

### Task list

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by jkcso** [comment]
@dduzgun-security What do you think?

**Review by dduzgun-security — CHANGES_REQUESTED** [review]
I think it would be good if we keep the `permissions` section since it restricts the default `GITHUB_TOKEN` [permission](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) used in the GitHub Action to only `contents: read`.

**Review by dduzgun-security — COMMENTED** [review]
Good idea to remove the `Check out code` step here. It would be nice to mention it as an item below in the **Solution Explanation**. Maybe something like: `# 7. Think about removing unnecessary steps in the GitHub Actions.`

> [!NOTE]  
> In this example the checkout step _copies_ the code in a GitHub runner but we actually don't need that step to run a simple `curl`. Removing it reduces the code of being _copied_ (gain in time) and therefore reduces the potential risk of code exposure if an attacker has access to the runner or finds a way to exploit the actions/checkout GitHub Action (gain in security).

**Comment by jkcso** [comment]
Thank you @dduzgun-security, feel free to suggest code + solution additions in the PR, make sure that you test too and I will review at the end to accept :-) Thanks!

**Comment by dduzgun-security** [comment]
👋 Hey @Debshibraj123, what do you think about the requested changes? I would have loved to suggest code change but it doesn't allow on removed lines of code.

**Comment by jkcso** [comment]
@Debshibraj123 any updates here please?

**Comment by nxhettry** [comment]
Guys i am a beginner, this suddenly appeared in my pull request. What is this? Someone tell me.

**Comment by nxhettry** [comment]
What are these comments and commits related to?

**Comment by jkcso** [comment]
@nxhettry don't worry my friend, nothing to do here for you

File: Season-2/Level-1/solution.yml [hunk 1]
```diff
@@ -11,15 +11,12 @@ on:
 jobs:
   jarvis:
     runs-on: ubuntu-latest
```
Comment (dduzgun-security) [inline Season-2/Level-1/solution.yml:13]:
```suggestion
    runs-on: ubuntu-latest
    permissions:
      contents: read
```

**Comment by dduzgun-security** [comment]
@jkcso If that sounds good with you, let's merge this and I'll address the [comment](https://github.com/skills/secure-code-game/pull/69#discussion_r1555106579) in another PR

**Review by jkcso — APPROVED** [review]

**Pull request merged**